frontend/src/metabase/querying/components/DatePicker/RelativeDatePicker/CurrentDatePicker/CurrentDatePicker.tsx

```diff
@@ -26,7 +31,7 @@ export function CurrentDatePicker({ value, onChange }: CurrentDatePickerProps) {
   };
 
   return (
-    <Stack p="md">
+    <Stack p={padded ? "md" : undefined}>
```

Comment:
Please remove `padded` from props and add padding where it's needed and where the component is used. Maybe with a `Box`

Reply:
lgtm